### PR TITLE
[Communication Chat] GA 2023-11-07 for inline image feature

### PR DESCRIPTION
**Pull request by palatter**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/communication/Azure.Communication.Chat/src/Models/CommunicationIdentifierModel.cs [hunk 1]
```diff
@@ -1,25 +1,12 @@
...
 {
-    // Excluded internal shared code.
-    [ExcludeFromCodeCoverage]
-    [CodeGenModel("CommunicationIdentifierModel")]
+    [CodeGenModel("CommunicationIdentifierModel", Usage = new string[] { "output" }, Formats = new string[] { "json" })]
     internal partial class CommunicationIdentifierModel
     {
-        /// <summary> Type of CommunicationIdentifierModel. </summary>
-        public CommunicationIdentifierModelKind? Kind { get; set; }
-
```
Comment (LuChen-Microsoft) [inline sdk/communication/Azure.Communication.Chat/src/Models/CommunicationIdentifierModel.cs:18]: why is this removed?

Comment (palatter) [reply sdk/communication/Azure.Communication.Chat/src/Models/CommunicationIdentifierModel.cs:18]: This was a work around as the 'kind' property in swagger wasn't correctly auto generated in the past. The current code generation correctly generates this though, and this manual workaround file is no longer needed

**Review by LuChen-Microsoft — APPROVED** [review]
